src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/help-loggingLevel.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    <p>Select the logging level option from the list</p>  
```

Comment:
For some reason, I don't like the "option from the list" part here. It doesn't seem like the other drop downs have it. Houman would probably have a good suggestion.